Failing `make repo-consistency`

### System Info

- `transformers` version: 4.46.0.dev0
- Platform: Linux-6.8.0-47-generic-x86_64-with-glibc2.39
- Python version: 3.12.3
- Huggingface_hub version: 0.25.1
- Safetensors version: 0.4.5
- Accelerate version: 1.0.1
- Accelerate config:    not found
- PyTorch version (GPU?): 2.4.1+cu121 (True)
- Tensorflow version (GPU?): 2.17.0 (False)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using distributed or parallel set-up in script?: <fill in>
- Using GPU in script?: <fill in>
- GPU type: NVIDIA GeForce RTX 3050 Laptop GPU

### Who can help?

@stevhliu @Cyrilvallez 

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

When checking repo-consistency by `makr repo-consistency` Test fails
```bash
$ make repo-consistency 
python utils/check_copies.py
python utils/check_modular_conversion.py
No differences found for src/transformers/models/llava_next_video/modeling_llava_next_video.py.
No differences found for src/transformers/models/llava_next_video/configuration_llava_next_video.py.
No differences found for src/transformers/models/gemma/modeling_gemma.py.
No differences found for src/transformers/models/gemma/configuration_gemma.py.
No differences found for src/transformers/models/gemma/tokenization_gemma.py.
No differences found for src/transformers/models/instructblipvideo/modeling_instructblipvideo.py.
No differences found for src/transformers/models/instructblipvideo/configuration_instructblipvideo.py.
No differences found for src/transformers/models/gemma2/modeling_gemma2.py.
No differences found for src/transformers/models/gemma2/configuration_gemma2.py.

Differences found between the generated code and src/transformers/models/glm/modeling_glm.py:

   1 --- src/transformers/models/glm/modeling_glm.py_generated
   2 +++ src/transformers/models/glm/modeling_glm.py
   3 @@ -38,7 +38,6 @@
   4  )
   5  from ...modeling_utils import PreTrainedModel
   6  from ...utils import (
   7 -    add_code_sample_docstrings,
   8      add_start_docstrings,
   9      add_start_docstrings_to_model_forward,
  10      is_flash_attn_2_available,
  11 @@ -1223,11 +1222,6 @@
  12          self.model.embed_tokens = value
  13  
  14      @add_start_docstrings_to_model_forward(GLM_INPUTS_DOCSTRING)
  15 -    @add_code_sample_docstrings(
  16 -        checkpoint=_CHECKPOINT_FOR_DOC,
  17 -        output_type=TokenClassifierOutput,
  18 -        config_class=_CONFIG_FOR_DOC,
  19 -    )
  20      def forward(
  21          self,
  22          input_ids: Optional[torch.LongTensor] = None,
Traceback (most recent call last):
  File "/home/h3110fr13nd/Desktop/dev/community/huggingface/transformers/utils/check_modular_conversion.py", line 76, in <module>
    raise ValueError("Some diff and their modeling code did not match.")
ValueError: Some diff and their modeling code did not match.
make: *** [Makefile:39: repo-consistency] Error 1
```

### Expected behavior

This should not fail.

--- a/src/transformers/models/glm/modeling_glm.py
+++ b/src/transformers/models/glm/modeling_glm.py
@@ -38,6 +38,7 @@ from ...modeling_outputs import (
 )
 from ...modeling_utils import PreTrainedModel
 from ...utils import (
+    add_code_sample_docstrings,
     add_start_docstrings,
     add_start_docstrings_to_model_forward,
     is_flash_attn_2_available,
@@ -52,6 +53,9 @@ if is_flash_attn_2_available():
     from ...modeling_flash_attention_utils import _flash_attention_forward
 
 from ...modeling_flash_attention_utils import _flash_attention_forward
+
+
+_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b-chat"
 
 
 class GlmRMSNorm(nn.Module):
@@ -1222,6 +1226,11 @@ class GlmForTokenClassification(GlmPreTrainedModel):
         self.model.embed_tokens = value
 
     @add_start_docstrings_to_model_forward(GLM_INPUTS_DOCSTRING)
+    @add_code_sample_docstrings(
+        checkpoint=_CHECKPOINT_FOR_DOC,
+        output_type=TokenClassifierOutput,
+        config_class=_CONFIG_FOR_DOC,
+    )
     def forward(
         self,
         input_ids: Optional[torch.LongTensor] = None,

--- a/src/transformers/models/glm/modeling_glm.py
+++ b/src/transformers/models/glm/modeling_glm.py
@@ -55,7 +55,7 @@ if is_flash_attn_2_available():
 from ...modeling_flash_attention_utils import _flash_attention_forward
 
 
-_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b-chat"
+_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b"
 
 
 class GlmRMSNorm(nn.Module):

--- a/src/transformers/models/glm/modular_glm.py
+++ b/src/transformers/models/glm/modular_glm.py
@@ -44,6 +44,8 @@ from ..phi3.modeling_phi3 import (
 from .configuration_glm import GlmConfig
 
 
+_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b-chat"
+
 logger = logging.get_logger(__name__)
 
 

--- a/src/transformers/models/glm/modular_glm.py
+++ b/src/transformers/models/glm/modular_glm.py
@@ -44,7 +44,7 @@ from ..phi3.modeling_phi3 import (
 from .configuration_glm import GlmConfig
 
 
-_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b-chat"
+_CHECKPOINT_FOR_DOC = "THUDM/glm-4-9b"
 
 logger = logging.get_logger(__name__)
 

